### ISSUE TITLE
DatePicker doesn't focus on picker panel on tab

### ISSUE THREAD

**Issue by wang-yikai** [ant-design/ant-design#48364]
### Reproduction link

[https://codepen.io/pen?&editors=001](https://codepen.io/pen?&editors=001)

### Steps to reproduce

Click on any of the DatePicker input field and press tab. It will focus on other DatePicker inputs and close the picker panel.

### What is expected?

In AntD v4 (https://codesandbox.io/p/sandbox/basic-antd-4-24-16-jzxz87), when the DatePicker input field is clicked and tab is pressed, the picker panel will be focused and you can use the arrow keys to select new dates.

### What is actually happening?

Other DatePicker input fields are being focused and the picker panel is closed when the tab key is pressed.

| Environment | Info |
| --- | --- |
| antd | 5.16.1 |
| React | 18 |
| System | MacOS |
| Browser | Chrome |

**Comment by zombieJ** [comment]
Yes. The panel accessible is removed since it's not well work with accessibility.

**Comment by austinjreid** [comment]
Is there any fix for this planned? Or a workaround?
